packaging/deb/control/prerm

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$1" = 'remove' ]
```

Comment:
Is this if statement necessary since this is in the --before-remove script?

Reply:
Yes, it's necessary since `prerm` is called as well on package upgrades for the old package with `upgrade` argument. And we do not want the service to be disabled on upgrades.

But we may want to have it stopped/restarted before/after upgrades? However, something for a new PR in case.